compressor/filters/css_default.py

```diff
@@ -104,3 +104,28 @@ def url_converter(self, matchobj):
 
     def src_converter(self, matchobj):
         return self._converter(matchobj, 2, "src=%s%s%s")
+
+
+class CssRelativeFilter(CssAbsoluteFilter):
+    """
+    Do similar to ``CssAbsoluteFilter`` URL processing
+    but replace ``settings.COMPRESS_URL`` prefix with '../../'.
+    """
+
+    def add_suffix(self, url):
+        url = super(CssRelativeFilter, self).add_suffix(url)
+        old_prefix = self.url
+        if self.has_scheme:
+            old_prefix = '{}{}'.format(self.protocol, old_prefix)
+        # One level up from 'css' / 'js' folder
+        new_prefix = ['..']
+        # N levels up from settings.COMPRESS_OUTPUT_DIR:
+        # - if it's 'CACHE' (default) then it'd be ['..'];
+        # - if it's 'CACHE/in/depth' then it'd be ['..', '..', '..'].
+        new_prefix += [
+            '..' for part in os.path.normpath(
+                settings.COMPRESS_OUTPUT_DIR
+            ).split(os.sep) if part
+        ]
+        new_prefix = '/'.join(new_prefix)
```

Comment:
maybe something like
```
# One level up from 'css' / 'js' folder
new_prefix = '..'
# N levels up from settings.COMPRESS_OUTPUT_DIR
levels = len(os.path.normpath(settings.COMPRESS_OUTPUT_DIR).split(os.sep))
new_prefix += (os.sep + '..') * levels
```
the `if part` shouldn't be necessary when using normpath, or did you add it for a specific reason?